### ISSUE TITLE
Month and year dropdowns not keyboard accessbile

### ISSUE THREAD

**Issue by vut18** [sproctor/php-calendar#143]
Month and year dropdowns are not accessible via keyboard; therefore, it is not Section 508 compliant.  Can you modify the code to make it accessible?

**Comment by sproctor** [comment]
This is fixed in v3 and will be available when that version is released
